tools/gromacs/check.xml

```diff
@@ -0,0 +1,249 @@
+<tool id="gmx_check" name="GROMACS check" version="@TOOL_VERSION@+galaxy@GALAXY_VERSION@" profile="@PROFILE@">
+    <description>to provide information about GROMACS input files</description>
+    <macros>
+        <token name="@GALAXY_VERSION@">0</token>
+        <token name="@PROFILE@">21.05</token>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        #if $input_file.calculate_select == "info_traj":
+            ln -s '$input_file.traj_file' './$input_file.traj_file.element_identifier';
+            gmx check
+                -f './$input_file.traj_file.element_identifier'
```

Comment:
```suggestion
            ln -s '$input_file.traj_file' './traj.$input_file.traj_file.ext';
            gmx check
                -f './traj.$input_file.traj_file.ext'
```
@Agnes-Petit I just noticed what you are doing here with `element_identifier`. The problem with that is that the tool will only work when the input datasets are named such that they end with the right extensions (to see what I mean, try renaming e.g. `npt.tpr` to `npt topology` and rerunning)

The suggested change should work no matter how the datasets are named. Could you add this and make similar changes in the rest of the command section as well?

Otherwise I think this should be ready to merge.

Reply:
I hadn't thought about this case. I have applied the changes you suggested.

Reply:
Thanks! Let's see if the tests pass